autoplex/fitting/common/utils.py

```diff
@@ -52,14 +52,14 @@
 )
 
 current_dir = Path(__file__).absolute().parent
-MLIP_PHONON_DEFAULTS_FILE_PATH = current_dir / "mlip-phonon-defaults.json"
-MLIP_RSS_DEFAULTS_FILE_PATH = current_dir / "mlip-rss-defaults.json"
+GAP_DEFAULTS_FILE_PATH = current_dir / "gap-defaults.json"
```

Comment:
as `MLIP_PHONON_DEFAULTS_FILE_PATH` has been renamed now to `GAP_DEFAULTS_FILE_PATH ` in this module, other modules where this is called also need to be changed. I found one place where it needs to be changed.

https://github.com/YuanbinLiu/autoplex_pub/blob/52731b1cbefd8a4546ee2b769430ff80ffb00868/autoplex/auto/phonons/flows.py#L10

Please also check if there are any other places where this is called.

Reply:
@QuantumChemist could you help us tomorrow to figure out how we should name this in the future. Especially because of your phonon workflow. Thanks!

Reply:
Yes, I've reverted the name back to the original, as I believe for potential fitting purposes, it shouldn't differentiate between RSS and phonon. The default settings for parameters should be consistent across modules. Since we plan to introduce new modules in the future, making the name more general seems appropriate. Of course, if @QuantumChemist has a better suggestion, I'd be happy to discuss more.

Reply:
See #149 for further discussion

Reply:
I would like to ask you to keep both files for now. The phonon workflow is finally running smoothly and I also think we might want different default settings in the future.

Reply:
Yes, for sure

Reply:
I have just reverted them to the @QuantumChemist's version. Let's keep it now and make the change later when necessary.

Reply:
Thanks!

Reply:
Based on @vlderinger 's reply here: https://github.com/autoatml/autoplex/issues/149#issuecomment-2463880454, I would advocate having two separate sets of MLIP parameter files. The user is able to adjust it to their needs anyway, but as per internal default settings, we should keep it separated.

Reply:
Yes, I think we reached consensus here :).

Reply:
So you mean having `mlip-phonon-defaults.json` and `mlip-rss-defaults.json`? That sounds good to me for now

Reply:
sorry, was late to the party 🙈